tests/unit/literaturesuggest/test_literaturesuggest_form.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+from inspirehep.modules.literaturesuggest.forms import LiteratureForm
+
+
+def test_degree_choices_contains_capitalized_phd():
+    """Ensure that lowercased degree type in the schema ends up with the
+    right capitalization in the form.
+    """
+    literature_form = LiteratureForm()
+
+    assert ('phd', 'PhD') in literature_form.degree_type.choices
```

Comment:
Going back to this, since PR https://github.com/inspirehep/inspire-next/pull/2290 could use a test similar to this.

I am actually not sure about this not being a correct test. The forms are generated from Python so asserting that a certain field has certain values ensures that those values will appear in the UI.